src/writeData/components/PluginCreateConfigurationOptions.tsx

```diff
@@ -55,7 +55,7 @@ const PluginCreateConfigurationOptionsComponent: FC<Props> = props => {
   }
 
   const handleSelectBucket = (bucket: Bucket) => {
-    const {orgID, id, name} = bucket
+    const {orgID, name, id} = bucket
```

Comment:
Flipped the order to match the order of the arguments in the function call a few lines below.